src/migrations/20240614000000-r10-clear-all-goals-and-objs.js

```diff
@@ -47,7 +47,11 @@ module.exports = {
         now() "updatedAt"
       FROM r_10_active_goals g
       LEFT JOIN "Users" u
-        ON u.name = 'Melissa Bandy-Ogden'
+        ON (
+          (u.id = 550 AND "phoneNumber" IS NOT NULL) --for non-prod
+          OR
+          (md5(u.name || ' 'bdjy34gg') = '7b1166c709d27ec6519b05c24373be6ai') --for prod
```

Comment:
it looks like there are too many single quotes

Reply:
Yeah, typo crept in somehow